src/app/legacy/containers/SocialEmbed/sourceHelpers.js

```diff
@@ -35,9 +35,16 @@ export const getProviderFromSource = source => {
 export const getIdFromSource = source => {
   const NO_ID = '';
   const provider = getProviderFromSource(source);
-  if (provider === PROVIDERS.TWITTER) {
-    const match = source.match(/\/status\/([0-9]+)/);
-    return match ? match[1] : NO_ID;
+  if (
+    provider === PROVIDERS.TWITTER ||
```

Comment:
not sure of our style guide, but could this be 

```
if ([PROVIDERS.TWITTER, PROVIDERS.YOUTUBE, PROVIDERS.INSTAGRAM].includes(provider)) {
```

Reply:
Yes, I think no harm in doing this. I will update soon